arch/arm/mach-zynqmp/Kconfig

```diff
@@ -167,6 +167,20 @@ config ZYNQMP_PSU_INIT_ENABLED
 	help
 	  Include psu_init to full u-boot. SPL include psu_init by default.
 
+config ZYNQMP_CMD_BOOTSLOT
+	bool "Provide command for switching A/B boot slots"
+	default y
+	help
+	  Command for switching A/B boot slots.
+
+config ZYNQMP_IMGSEL_PERSISTENT_REGISTER
+	default 0x100000
+	hex
+	help
+	  This option provides an address of persistent register offset,
+	  which is used by imgsel tool for detecting current bootable
+	  slot (A/B).
```

Comment:
Would be nice to extend this a bit more to point to imgsel as that is where this gets configured.